Implement new RESTORE options

michael-grunder opened the pull request:
Add the new RESTORE options REPLACE, ABSTTL, FREQ <freq> and IDLETIME <idletime>

Fixes #1410

yatsukhnenko commented on redis_commands.c:
```diff
@@ -2324,6 +2331,98 @@ int redis_lcs_cmd(INTERNAL_FUNCTION_PARAMETERS, RedisSock *redis_sock,
+        return;
+
+    ZEND_HASH_FOREACH_STR_KEY_VAL(ht, key, zv) {
+        if (key) {
```
I suggest to add `ZVAL_DEREF` here or even on previous line

michael-grunder committed:
Implement new RESTORE options

Add the new RESTORE options REPLACE, ABSTTL, FREQ <freq> and IDLETIME <idletime>

Fixes #1410
